README.md

```diff
@@ -13,3 +14,28 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+1:
+
+- Actual performance of programs/algorithms can also rely heavily on the hardware being used to run them. With how performative computers are in this day, you might not even notice a difference, depending on the input size and the optimization. Optimizations that I am thinking of in this context is how the program is implemented. A specific example in C++ could be using compare() versus ==, where compare() takes significantly longer than == because of how it actually performs the comparison between strings.
+  
+- The difference in performance that we are sometimes talking about with asymptotic analysis can to be very minuscule between different algorithms. With this in mind, there are many cases where the difference in performance can be functionally the same, within margin of error, rounding based on significant figures of runtime calculation, etc. For example, a specific algorithm has an average case runtime of n^2logn while another has one of n^2. At an input size of 10, which is a reasonable input size for basic testing when coding, the first algorithm would have a runtime of 100 seconds and the second would have the same runtime.
```

Comment:
...but asymptotic analysis doesn't tell you anything about the runtime for a specific input size.